documentation/LLC-quickstart.md

```diff
@@ -0,0 +1,401 @@
+    
+    ---  
+  
+1. **edit rush.json**  
+    As the libraries in this azure-sdk-for-js repository are managed by rush, you need to add an entry in rush.json under projects section to make sure it works. For example:
+    ```
+        {
+          "packageName": "@azure-rest/purview-account",
+          "projectFolder": "sdk/purview/purview-account-rest",
+          "versionPolicyName": "client"
+        },
+    ```
+    Here you also need to replace the `packageName`, `projectFolder` into your own services'.
+    
+    ---  
+    **NOTE** 
+    
+    About the `versionPolicyName`, if the library you are working on is for data-plane, then it should be `client`, if the library you are working on is for control plane, then it should be `mgmt`.  
+    
+    ---  
+
+1. **run autorest to generate the SDK**  
+
+    Now you can run this command in swagger folder you just created. 
+    ```shell
+    autorest --typescript ./README.md
+    ```
+    After this finishes, you will see the generated code in src folder which are in the same level with the swagger folder.   
+    
+    
+    After that, you can get a workable package, and run the following commands to get a artifact if you like.
+    
+    ```shell
+    rush update
+    rush build -t <your-package-name>
+    cd <your-sdk-folder>
+    rushx pack
+    ``` 
+    But we still need to add some tests for it. 
+# How to write test for RLC
+In order to release it, we need to add some tests for it to make sure we are delivering high quality packages. but before we add the test, we need to manual change a few things to make the test framework works. 
+1. **update package.json file**  
+    Currently the generated will skip the actual test step. you should change it to make sure it works. 
+    First, change the `scripts` section from 
+    ~~~
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "test:node": "echo skipped",
+        "test:browser": "echo skipped",
+        "unit-test": "echo skipped",
+        "unit-test:node": "echo skipped",
+        "unit-test:browser": "echo skipped",
+        "integration-test:browser": "echo skipped",
+        "integration-test:node": "echo skipped",
+        "integration-test": "echo skipped",
+    ~~~
+    into 
+    
+    ~~~
+        "test": "npm run clean && npm run build:test && npm run unit-test",
+        "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
+        "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
+        "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+        "unit-test:node": "cross-env TEST_MODE=playback mocha -r esm --require ts-node/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace \"test/{,!(browser)/**/}*.spec.ts\"",
+        "unit-test:browser": "cross-env TEST_MODE=playback karma start --single-run",
```

Comment:
please remove TEST_MODE=playback from the unit-test scripts. Otherwise the will not be able to record